.gitignore

```diff
@@ -7,6 +7,10 @@
 /cmake
 /devdoc/~$o_requirements.docm
```

Comment:
This can also be removed.

Reply:
@schultetwin1 I'll take care of this one.